habit/src/containers/signup/SignupScreen.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { View, Text} from 'react-native'
+import CustomTextInput from '../../components/TextInput'
+import Header from '../../components/Header'
+import * as CONSTANTS from '../../utils/string_constants'
+import {Button} from 'react-native-elements'
+import commonStyles from '../../styles/common'
+
+export default function Signup(props){
+
+    const [email, setEmail] = React.useState('');
+    const [password, setPassword] = React.useState('');
+    const [gender, setGender] = React.useState('');
+    const [firstname, setFirstname] = React.useState('');
+    const [lastname, setLastname] = React.useState('');
+
+    const handleSignup = () => {
+        //logic
+        console.log("email", email)
+    }
+
+    return(
+        <View style={commonStyles.page}>
+            {/* TODO AYUSHI- 16-08-2020 add an image here and a pretty background */}
+            <Header onPress={() => props.navigation.goBack()}/>
+            <Text>Sign up</Text>
+            <CustomTextInput 
+                placeholder = {CONSTANTS.FIRSTNAME}
```

Comment:
Do we need to ask the user these two separately??

Ideally, we should separate it out ourselves with the fist/last space

Reply:
no, this should be like this only